pkg/controller/node/nodecontroller.go

```diff
@@ -145,38 +150,43 @@ func (nc *NodeController) Run(period time.Duration) {
 	}, nc.nodeMonitorPeriod)
 
 	go util.Forever(func() {
-		nc.podEvictor.TryEvict(func(nodeName string) { nc.deletePods(nodeName) })
+		nc.podEvictor.Try(func(value TimedValue) (bool, time.Duration) {
```

Comment:
Yes, TimedValue as a type is OK, I'm saying that the variable name can be more descriptive. At first I wasn't certain if Value stores Node name or Pod name.